ts/defs.bzl

```diff
@@ -397,10 +399,26 @@ def ts_project(
             **common_kwargs
         )
 
+        # Users should build this target to get a failed build when typechecking fails
+        native.filegroup(
+            name = transitive_typecheck_target_name,
+            srcs = [name],
+            output_group = "transitive_typecheck",
+            **common_kwargs
```

Comment:
Does this also have to be manual to avoid `build //...` triggering all transitive `{name}_transitive_typecheck` targets to build?